engine/Assets/Scripts/Drivers/RotationalDriver.cs

```diff
@@ -84,23 +84,23 @@ public enum RotationalControlMode {
             get => _motor;
             set {
                 _motor = value;
-                SimulationPreferences.SetRobotJointMotor((_simObject as RobotSimObject).MiraGUID, Name, _motor);
+                SimulationPreferences.SetRobotJointMotor((_simObject as RobotSimObject)!.MiraGUID, Name, _motor);
             }
         }
         private HingeJoint _jointA;
         public HingeJoint JointA => _jointA;
         private HingeJoint _jointB;
         public HingeJoint JointB => _jointB;
 
-        private Rigidbody _rbA = null;
+        private Rigidbody? _rbA;
```

Comment:
Is there the possibility that this gets assigned a null value later on?

Reply:
Its a matter of removing warnings